test/src/test/java/hudson/model/ParameterDefinitionTest.java

```diff
@@ -0,0 +1,147 @@
+package hudson.model;
+
+import hudson.util.Secret;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+public class ParameterDefinitionTest {
```

Comment:
Since this is a unit test (no `JenkinsRule` dep), it can be moved into the `core` module.

Reply:
@jglick , should I move tests to `core/src/test/java`?

Reply:
Yes, exactly.

Reply:
Done